modules/preview-react/side-panel/lib/SidePanelToggleButton.tsx

```diff
@@ -0,0 +1,78 @@
+/** @jsxRuntime classic */
```

Comment:
I wonder why we need these in some places but not others.

Reply:
We need it when using the `css` prop. This is one of the reasons we avoid it.